src/component/App.js

```diff
@@ -1,20 +1,136 @@
-import React from 'react';
+import React, { Component } from 'react';
 import Display from './Display';
 import Bpanel from './Bpanel';
 import '../App.css';
+import calculate from '../logic/calculate';
+import operate from '../logic/operate';
+import Particleswrap from './Particlewrap';
 
-// import calculate from '../logic/calculate'; //commented for passing linter errors
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: '0',
+      next: '',
+      operation: null,
+    };
+  }
 
-const App = () => (
-  <div>
-    <div className="container-fluid">
-      <div className="row">
-        <div className="col-4 mx-auto caluatorBody shadow-lg mt-5 p-3">
-          <Display />
-          <Bpanel />
+  initialState = (total, next, operation) => ({ total, next, operation })
+
+  handleClick = (buttonName) => {
+    if (['-', '+'].includes(buttonName)) {
+      this.setState({
+        operation: buttonName,
+      }, () => {
+        calculate(this.state, buttonName);
+        const { total, next, operation } = this.state;
+        const numberONe = parseInt(total, 10);
+        const numberTwo = next === '' && (buttonName === '+' || buttonName === '-') ? '0' : parseInt(next, 10);
+        this.setState({
+          total: operate(numberONe, numberTwo, operation),
+          next: '',
+        });
+      });
+    } else if (buttonName === '=') {
+      const { total, next, operation } = this.state;
+      this.setState({
+        total: operate(total, next, operation),
+      }, () => {
+        calculate(this.state, buttonName);
+        this.setState(this.initialState(total, '', null), () => {
+          this.setState({
+            total: operate(total, next, operation),
+          }, () => { });
+        });
+      });
+    } else if (buttonName === '+/-') {
+      this.setState((prevState) => ({
+        total: calculate(prevState, buttonName).total,
+        next: calculate(prevState, buttonName).next,
+      }));
+    } else if (buttonName === 'AC') {
+      this.setState(calculate(this.initialState, buttonName), () => { });
+    } else if (['x', '÷', '%'].includes(buttonName)) {
+      this.setState({
+        operation: buttonName,
+      }, () => {
```

Comment:
You have not handled division by ZERO 😊
These section too, should normally be inside `operate.js` where all operations (`logic/` should be stored